integration_test/src/test/java/com/newrelic/telemetry/MetricApiIntegrationTest.java

```diff
@@ -18,11 +18,7 @@
 import com.newrelic.telemetry.exceptions.RetryWithBackoffException;
 import com.newrelic.telemetry.exceptions.RetryWithRequestedWaitException;
 import com.newrelic.telemetry.exceptions.RetryWithSplitException;
-import com.newrelic.telemetry.metrics.Count;
-import com.newrelic.telemetry.metrics.Gauge;
-import com.newrelic.telemetry.metrics.MetricBatchSender;
-import com.newrelic.telemetry.metrics.MetricBuffer;
-import com.newrelic.telemetry.metrics.Summary;
+import com.newrelic.telemetry.metrics.*;
```

Comment:
much as it pains me to say it, I think our coding standards don't allow wildcard imports.

Reply:
ref here: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports